WordPress/gutenberg
# CustomSelectControl: Add flag for larger default size

Part of #39397

## What?

Adds an internal `__next36pxDefaultSize` feature flag prop to introduce the larger default size for `CustomSelectControl`.

## Why?

In preparation for standardizing the default sizes to 36px height.

## How?

Following the procedure outlined in #39397, conditional styles have been added to enlarge the height and paddings. The font size is also enlarged to be consistent with `SelectControl`.

The inconsistent chevron (#39400) will be addressed separately.

@jasmussen How do the paddings feel?

## Testing Instructions

1. `npm run storybook:dev` and see the stories for `CustomSelectControl`.
2. From the Controls tab, toggle the `__next36pxDefaultSize` prop.

## Screenshots

### `__next36pxDefaultSize` = false
<img src="https://user-images.githubusercontent.com/555336/157986345-3a7dee16-ff2b-45dc-b05e-fedbb9139ace.png" alt="Current default size" width="600">

### `__next36pxDefaultSize` = true

_Updated with 16px padding:_

<img width="600" alt="Larger default size" src="https://user-images.githubusercontent.com/555336/158231916-a3957ade-c5ad-4786-b5b3-81b1b49465b0.png">
